operator/src/main/resources/scripts/utils.sh

```diff
@@ -742,7 +744,6 @@ function checkAuxiliaryImage() {
       trace "Auxiliary Image: End of '${out_file}' contents"
     done
     [ "${severe_found}" = "true" ] && return 1
-    rm -fr $AUXILIARY_IMAGE_PATH/auxiliaryImageLogs
     [ -z "$(ls -A $AUXILIARY_IMAGE_PATH)" ] \
```

Comment:
This check is no longer valid now that auxiliaryImageLogs is always going to be there.

Reply:
Maybe have `"$(ls -A $AUXILIARY_IMAGE_PATH 2>/dev/null | grep -v auxiliaryImageLogs)"`